storage/remote/write_handler_test.go

```diff
@@ -551,27 +539,43 @@ func TestOutOfOrderHistogram_V1Message(t *testing.T) {
 }
 
 func TestOutOfOrderHistogram_V2Message(t *testing.T) {
-	payload, _, _, err := buildV2WriteRequest(nil, []writev2.TimeSeries{{
-		LabelsRefs: []uint32{0, 1},
-		Histograms: []writev2.Histogram{HistogramToV2HistogramProto(0, &testHistogram), FloatHistogramToV2HistogramProto(1, testHistogram.ToFloat(nil))},
-	}}, []string{"__name__", "metric1"}, nil, nil, nil, "snappy") // TODO(bwplotka): No empty string!
-	require.NoError(t, err)
+	for _, tc := range []struct {
+		Name      string
+		Timestamp int64
+	}{
+		{
+			Name:      "historic",
+			Timestamp: 0,
+		},
+		{
+			Name:      "future",
+			Timestamp: math.MaxInt64,
+		},
+	} {
+		t.Run(tc.Name, func(t *testing.T) {
+			payload, _, _, err := buildV2WriteRequest(nil, []writev2.TimeSeries{{
+				LabelsRefs: []uint32{0, 1},
+				Histograms: []writev2.Histogram{writev2.FromIntHistogram(0, &testHistogram), writev2.FromFloatHistogram(1, testHistogram.ToFloat(nil))},
+			}}, []string{"__name__", "metric1"}, nil, nil, nil, "snappy") // TODO(bwplotka): No empty string!
```

Comment:
nit: what does that TODO mean?

Reply:
heh, probably more of the note. It just means that PRW 2.0 symbols have to have the empty string, but nothing to do here. Will delete.